src/Components/UI/Navigation/DesktopNavigation/DesktopNavigationBar.css

```diff
@@ -6,10 +6,10 @@
 .profile-hover-menu {
   background-color: var(--primary-color);
   width: 8.7rem;
-  height: 11rem;
+  height: 7rem;
   position: absolute;
   top: 4rem;
-  right: 2.5rem;
+  right: 10px;
   border-radius: 4px;
   padding: 12px;
```

Comment:
use rems as the padding size is more than 10px

Reply:
I am using the default root font size of 16px. If I converted it into the rems, then it will be 0.6rem.